pkgs/by-name/be/bend/package.nix

```
@@ -0,0 +1,41 @@
+{ rustPlatform, fetchCrate, fetchFromGitHub, lib, stdenv }:
+
+let
+  hvm = rustPlatform.buildRustPackage rec {
+    pname = "hvm";
+    version = "2.0.12";
+    src = fetchCrate {
+      inherit pname version;
+      hash = "sha256-/55SK/5zBKXmucRQPoYt/8IHxisQlOxNEVMAZVMtCNI=";
+    };
+    RUSTC_BOOTSTRAP = true;
+    cargoHash = "sha256-9U8Y0KaQHIfOZnCKbl94VvjS/7Qmi6UnKMDZDTXcye0=";
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "Bend";
+  version = "0.2.9";
+
+  src = fetchFromGitHub {
+    owner = "HigherOrderCO";
+    repo = pname;
+    rev = "refs/tags/${version}";
+    hash = "sha256-MEfB2SBJN7uEwfZGoEL7DQXsr1fccdZyGyzHtNv9wow=";
+  };
+
+  cargoHash = "sha256-+i+Y3MgCBVN3REmPwAjm2SiF9FJ0i05czmPKB8JtAFM=";
+
+  RUSTC_BOOTSTRAP = true;
+
+  nativeBuildInputs = [ hvm ];
+
+  meta = with lib; {
+    description =
+      "Bend is a massively parallel, high-level programming language.";
```

```suggestion
      "Bend is a massively parallel, high-level programming language";
```

From [pkgs/README.md](https://github.com/NixOS/nixpkgs/blob/baf54cdc1df81f72081e35e82a17fc01bd605aa2/pkgs/README.md?plain=1#L372):

* `meta.description` must: 
  * Not end with a period (or any punctuation for that matter).